docs/api/ref/schemas/product_nutrition_properties.yaml

```diff
@@ -0,0 +1,46 @@
+type: object
+properties:
+  preparation:
+    type: string
+    enum:
+      - as_sold
+      - prepared
+    description: |
+      Indicates whether the nutrition values refer to the product *as_sold* or *prepared*.
+
+      The preparation state affects nutrient values.
+  per:
+    type: string
+    enum:
+      - 100g
+      - 100ml
+      - serving
+    description: |
+      The nutrition data on the package can be per serving, per 100g or per 100ml.
+
+      This is essential to understand if values in the `nutrients` object apply for a serving, for 100g or for 100ml.
+  per_quantity:
+    type: integer
```

Comment:
This should be a float. It's rare, but a few products have quantities with floating point. e.g. this old spray has a serving quantity of 0.25 ml: https://world.openfoodfacts.org/product/4250519660295/2-calories-oil-spray-esn